query/src/main/java/org/infinispan/query/dsl/embedded/impl/BaseJPAFilterIndexingServiceProvider.java

```diff
@@ -53,7 +52,8 @@
    private ClusteringDependentLogic clusteringDependentLogic;
 
    @Inject
-   protected void injectDependencies(CacheNotifier cacheNotifier, ClusteringDependentLogic clusteringDependentLogic, EncoderRegistry encoderRegistry) {
+   @SuppressWarnings("unused")
```

Comment:
This shouldn't be necessary, you can configure IntelliJ to consider all methods with the `@Inject` annotation as used.